src/components/form-helpers/form-helpers.js

```diff
@@ -530,6 +530,18 @@ export const NumberField = ({ required, ...rest }) => (
   <Field type="number" validate={createRequiredValidation(required)} {...rest} />
 );
 
+export const SwitchField = ({ id, name, text, ...rest }) => (
+  <>
+    <div className="dp-switch">
+      <Field type="checkbox" id={id || name} name={name} {...rest} />
+      <label htmlFor={id || name}>
+        <span />
+      </label>
+    </div>
+    <label htmlFor="switch">{text}</label>
```

Comment:
Me parece que no está bien hardcodear `switch`, no? no debería ser así?

```suggestion
    <label htmlFor={id || name}>{text}</label>
```

Reply:
@GusBaamonde, creo que hay un bug en el código de ejemplo de la librería:

![2021-07-07_10-35-25 (1)](https://user-images.githubusercontent.com/1157864/124768820-5cd68c00-df0f-11eb-9ad1-8bb1d024a5b0.gif)

Reply:
uups...